vlib/v/checker/tests/struct_field_init_and_default_is_map_err.out

```diff
@@ -0,0 +1,14 @@
+vlib/v/checker/tests/struct_field_init_and_default_is_map_err.vv:7:25: error: cannot copy map: call `move` or `clone` method (or use a reference)
+    5 | struct Foo {
+    6 | mut:
+    7 |     field map[string]int = const_map
```

Comment:
I think that only `clone` should be suggested here, because `move` will make the `const_map` contain invalid data.
Consider the case, where you have several fields, all doing `= const_map.move()` for their default. One of them will get the data of the original map, the rest will have rubbish.

Reply:
hmm...  yes.